src/components/navbar/navbar.css

```diff
@@ -0,0 +1,126 @@
+/* @import url(https://img.icons8.com/ios-glyphs/30/000000/frequency-f.png); */
+:root {
```

Comment:
don't repeat same thing in different files. you can use color variable in separate file and use it every place